index.html

```diff
@@ -105,5 +107,219 @@ <h1 class="title">I’m Michael Glad to see you!</h1>
+        <p class="project-title">Multi-Post Stories</p>
+        <div class="project-details">
+          <span class="client-label">canopy</span>
+          <span class="middot">&middot;</span>
+          <span class="role-label">back end dev</span>
+          <span class="middot">&middot;</span>
+          <span class="year-label">2015</span>
+        </div>
+        <div class="project-desc">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </div>
+        <ul class="tags">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+        </ul>
+        <button type="button" class="primary-btn">See Project</button>
+      </div>
+
+      <div class="projects">
+        <img src="images/snapshoot-project-3.png" alt="snapshoot-project-3" />
+        <p class="project-title">Tonic</p>
+        <div class="project-details">
+          <span class="client-label">canopy</span>
+          <span class="middot">&middot;</span>
+          <span class="role-label">back end dev</span>
+          <span class="middot">&middot;</span>
+          <span class="year-label">2015</span>
+        </div>
+        <div class="project-desc">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </div>
+        <ul class="tags">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+        </ul>
+        <button type="button" class="primary-btn">See Project</button>
+      </div>
+
+      <div class="projects">
+        <img src="images/snapshoot-project-4.png" alt="snapshoot-project-4" />
+        <p class="project-title">Multi-Post Stories</p>
+        <div class="project-details">
+          <span class="client-label">canopy</span>
+          <span class="middot">&middot;</span>
+          <span class="role-label">back end dev</span>
+          <span class="middot">&middot;</span>
+          <span class="year-label">2015</span>
+        </div>
+        <div class="project-desc">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </div>
+        <ul class="tags">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+        </ul>
+        <button type="button" class="primary-btn">See Project</button>
+      </div>
+    </section>
+    <section class="about-section">
+      <div class="top-section">
+        <p class="title">About Myself</p>
+        <p class="outro">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <div class="social-media">
+          <p class="supporting-text">Let's connect</p>
```

Comment:
- [ ] This also should be wrapped in a `hx` tag, kindly change that. 😄

Reply:
Changed it to an `h3` tag